.github/workflows/build.yml

```diff
@@ -51,3 +51,48 @@ jobs:
         pod install
         ../scripts/xcodebuild-ios.sh TemplateExample.xcworkspace build
       working-directory: template-example
+  Android-macOS:
+    runs-on: macos-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8    
+    - name: Install
+      run: |
+        yarn
+      working-directory: example
+    - name: Build
+      run: |
+        set -eo pipefail
+        yarn build:android
+        sh gradlew clean build check test
+      working-directory: example
+  Android-windows:
+    runs-on: windows-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8    
+    - name: Set up Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.9.1'  
+    - name: Install
+      run: |
+        yarn
+      working-directory: example
+    - name: Build
+      shell: cmd
+      run: yarn build:android                
+      working-directory: example
+    - name: Build android
```

Comment:
Github Actions, or CMD, ignores the second line of the script. That's the reason why ended-up having two separate blocks.

Reply:
Why not run Bash then?

Reply:
Oh wow, it actually worked. Should have done that in the first place.. I will try to rework two definitions into one, since both of them follow the same steps.

Reply:
Holy crap, the single job definition worked too :D

Reply:
Yeah, this looks really nice now.